### ISSUE TITLE
Figure 4-28 description is wrong

### ISSUE THREAD

**Issue by camiteca** [dotnet/docs#9975]
The description says:

&gt;Figure 4-28: Enabling Docker Compose support on a .NET Framework web app project in Visual Studio 2017

I've tried to add support to a .net framework project and the option isn't there in VS2017 15.9.5

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: bcf20f50-2d15-1744-386d-1d05ff9931fb
* Version Independent ID: a3c5626b-bbf1-dae0-5cea-0e1135957258
* Content: [Visual Studio Tools for Docker on Windows](https://docs.microsoft.com/en-us/dotnet/architecture/containerized-lifecycle/design-develop-containerized-apps/visual-studio-tools-for-docker)
* Content Source: [docs/architecture/containerized-lifecycle/design-develop-containerized-apps/visual-studio-tools-for-docker.md](https://github.com/dotnet/docs/blob/master/docs/architecture/containerized-lifecycle/design-develop-containerized-apps/visual-studio-tools-for-docker.md)
* Product: **dotnet**
* Technology: **dotnet-ebooks**
* GitHub Login: @nishanil
* Microsoft Alias: **nanil**

**Comment by mvelosop** [comment]
Hi @camiteca, thanks for your feedback.

However, there's no error here, as the support for Docker is only available for **ASP.NET Core** projects, not for **ASP.NET** (.NET Framework).

That's actually mentioned in the Microservices guide here:

![image](https://user-images.githubusercontent.com/7357806/67858932-4d842600-fb12-11e9-8127-1efb0f8fe9e6.png)

And in the [Visual Studio Container Tools with ASP.NET Core page](https://docs.microsoft.com/aspnet/core/host-and-deploy/docker/visual-studio-tools-for-docker):

![image](https://user-images.githubusercontent.com/7357806/67859137-b7043480-fb12-11e9-9727-7d267357b31a.png)

I regret not being of much help here, but that's just the way it is.

**Comment by mvelosop** [comment]
Closing this issue now but feel free to comment, will reopen if needed.

**Comment by mvelosop** [comment]
Oops, I'm sorry @camiteca, I got confused with the image numbering. It looks like it changed from the version you're using.

I tried what you mentioned in VS 15.9.13 and I do see the option shown (currently) in figure 4-33.

I guess the clarifications I highlighted mean about supporting Linunx "**and**" Windows containers.

One important caveat, it has to be an **ASP.NET** application.

Hope this helps.
